.github/workflows/swift-toolchain.yml

```diff
@@ -2118,9 +2118,119 @@ jobs:
           symbolsFolder: ${{ github.workspace }}/BinaryCache
           searchPattern: '**/*.exe'
 
+  debugging_tools:
+    name: Debugging Tools
+    needs: [context, compilers, devtools, sdk]
+    runs-on: ${{ needs.context.outputs.windows_build_runner }}
+    env:
+      TOOLCHAIN_PATH: ${{ github.workspace }}/BuildRoot/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr/bin
+      # Where to store new binaries.
+      DEBUGGING_TOOLS_DIR: ${{ github.workspace }}/BuildRoot-DebuggingTools/Library/Developer/Toolchains/unknown-Asserts-development.xctoolchain/usr
```

Comment:
This is different from the other portions of the workflow. Should we try to keep this uniform and then do a change universally? How would this scale when you have $n$ repositories to build in a stage? (e.g. devtools).

Reply:
Hmm, I'm not sure what you're asking: are you saying that we should use `BuildRoot-DevTools` here instead of `BuildRoot-DebuggingTools`, or that this particular job should not store all of the tools it builds in a single common root directory, specified by this env var?

Reply:
I'm simply pointing out the use of the environment variable is different.

Reply:
Ah, I see. Doing it later, all at once sounds fine with me.